pkg/logmgr/logmgr.go

```diff
@@ -255,9 +255,16 @@ func (lm *LogManager) rotateLogs() {
 		lm.op.Debugf("logrotate is not defined. Skipping.")
 		return
 	}
-	if err := exec.Command(LogRotateBinary, configFile).Run(); err == nil {
+	output, err := exec.Command(LogRotateBinary, configFile).CombinedOutput()
+	if err == nil {
+		if len(output) > 0 {
+			lm.op.Debugf("Logrotate output: %s", output)
```

Comment:
@hmahmood It is useful during debug session when log rotate happens. I add a "-v" key to see what happened exactly.